api/src/main/java/org/openmrs/module/isanteplusreports/RegisterReports.java

```diff
@@ -23,13 +24,14 @@ public void registerReports() {
 			String version = ModuleFactory.getModuleById("isanteplusreports").getVersion();
 			String oldversion = Context.getAdministrationService().getGlobalProperty("reports.moduleVersion");
 
-			if (!version.equals(oldversion)) {
+			//if (!version.equals(oldversion)) {
```

Comment:
Why would you want to change this? I think it was put in place by design. 
It is OK to comment it during development, but it shouldn't be part of the PR

Reply:
Yes @moshonk , ideally i comment it just for development . In the final PR i revert it

Reply:
Hello Mose, @moshonk , did that PR take into account the indicator desagragation as well

Reply:
oh ,it actually doesnt (i havent taken it into account) , @moshonk   has just given me that comment in slack. 
I will re-work on it